### ISSUE TITLE
Getting the following error

### ISSUE THREAD

**Issue by paulsschwarz** [calcinai/xero-php#1]
`Parsing method not implemented for [text/html]`

The readme doesn't indicate that a calling `execute()` is necessary for a load. However, by following the code, I think it is. But when I do that, I think the computed URL is not correct because it ends up loading an IIS server 404 page on the Xero server.

**Comment by calcinai** [comment]
You're correct - I need to make some better docs.  The reason you need to call ```->execute()``` is to allow you to chain the load command (with a limit or a where), before executing.

Which resource are you trying to access? I just did a quick test as follows:

```php
$xero = new PrivateApplication($config);
$contact = $xero->load('Accounting\\Contact')->execute();
```

The above is successful, is that similar to what you're doing?  It's possible one of the parsed endpoints in the docs is not correct.

**Comment by paulsschwarz** [comment]
Thanks for the reply.

The end point I was trying was `$xero->load('Accounting\\Organisation')->
execute();`

**Comment by calcinai** [comment]
This should be resolved with v1.0.5, composer update it and let me know how you go.
